src/newlines.cpp

```diff
@@ -2612,6 +2612,16 @@ void newlines_cleanup_braces(bool first)
       if (pc->type == CT_IF)
       {
          newlines_if_for_while_switch(pc, cpd.settings[UO_nl_if_brace].a);
+         tmp = chunk_get_next_type(pc, CT_SPAREN_CLOSE, pc->level);
+         if (tmp != nullptr)
+         {
+            prev = chunk_get_prev(tmp);
+            if (prev != nullptr)
+            {
+               // Issue #1139
+               newline_iarf_pair(prev, tmp, cpd.settings[UO_nl_before_closing_paren].a);
+            }
+         }
```

Comment:
As far as I see the `nl_before_closing_paren` option is only working on CT_IF chunks? Unless you plan to extend this I'd rename that option to `nl_before_if_closing_paren`.